binary/test/Test/Cardano/Binary/Helpers.hs

```diff
@@ -179,72 +168,6 @@ extensionProperty = forAll @a (arbitrary :: Gen a) $ \input ->
     (encoded :: a) = unsafeDeserialize (serialize u)   -- Step 3
   in encoded === input
 
---------------------------------------------------------------------------------
--- Message length
---------------------------------------------------------------------------------
-
-msgLenLimitedCheck :: ToCBOR a => Limit a -> a -> Property
```

Comment:
Can you check whether this stuff has been translated into `hedgehog` tests in `cardano-crypto-tests`? We should have these tests, even if they're currently unused.

Reply:
https://github.com/input-output-hk/cardano-ledger/blob/e2d4663f2251ef76bae94e48fef910b8f5f867bd/crypto/test/Test/Cardano/Crypto/Limits.hs#L59